src/language-graphql/parser-graphql.js

```diff
@@ -34,11 +34,19 @@ function removeTokens(node) {
   return node;
 }
 
+function fallbackParser(parse, source) {
+  try {
+    return parse(source, { allowLegacySDLImplementsInterfaces: false });
+  } catch (_) {
+    return parse(source, { allowLegacySDLImplementsInterfaces: true });
```

Comment:
It'd be great if we could detect which style the code was originally written so we could support both styles, and we can with this code.

But I don't think it's worth the trouble though.

Reply:
I would agree with that.